packages/react-instantsearch-core/src/connectors/__tests__/connectNumericMenu.js

```diff
@@ -268,6 +268,8 @@ describe('connectNumericMenu', () => {
             { label: '2', start: 400 },
             { label: '3', end: 200 },
             { label: '4', start: 100, end: 200 },
+            { label: '5', start: 300 },
+            { label: '6', start: 300, end: 300 },
```

Comment:
I think I'd rather see a separate test for this too, so it can easily be ported to InstantSearch.js (where it already works)

Reply:
This test specifically verifies the `noRefinement` value for each item range, which is not implemented in IS.js, so I don't see how it could be ported. Do you have something specific in mind?